activefires_pp/post_processing.py

```diff
@@ -357,6 +316,7 @@ def __init__(self, configfile, shp_borders, shp_mask, regional_filtermask=None):
         self.output_topic = self.options['publish_topic']
         self.infile_pattern = self.options.get('af_pattern_ibands')
         self.outfile_pattern_national = self.options.get('geojson_file_pattern_national')
+        self.outfile_pattern_national_sweref99 = self.options.get('geojson_file_pattern_national_sweref99')
```

Comment:
Is there a way to make this less specific? From what I understand the idea is to switch ellipsoids, so what about passing the desired output ellipsoid as an option instead?
Regarding the output file, we could imagine having in the config a list of output patterns with optional ellipsoid for example?

Reply:
Ok, yes I see, I think I already contemplated something like this, but went for the simple and fast (though less flexible) solution. Let's discuss...

Reply:
Ok, done. Ok?